test/test.hs

```diff
@@ -77,11 +77,11 @@ import EVM.Expr (maybeLitByteSimp)
 
 testEnv :: Env
 testEnv = Env { config = defaultConfig {
-  dumpQueries = False
-  , dumpExprs = False
-  , dumpEndStates = False
-  , debug = False
-  , dumpTrace = False
+  dumpQueries = True
+  , dumpExprs = True
+  , dumpEndStates = True
+  , debug = True
+  , dumpTrace = True
```

Comment:
Did you mean to change the default config?

Reply:
Shit not at all. Sorry....